### PR TITLE
Update LEOX lxt-010s-h wiki

### PR DESCRIPTION
**Pull request by stich86**

### CODE REVIEW

File: _ont/ont-leox-lxt-010s-h.md [hunk 1]
```diff
@@ -12,27 +12,65 @@ parent: Leox
...
-| Web Gui     |                                      |
+| Web Gui     | ✅                                   |
 | Telnet      | ✅ user `leox`, password `leolabs_7` |
 | Form Factor | miniONT SFP                          |
 
 
 ## List of software versions
-## List of partitions
-## List of firmwares and files
+# List of partitions
```
Comment (simonebortolin) [inline _ont/ont-leox-lxt-010s-h.md:27]: List of partition are (in all other pages) a second level header, use `##` inside `#`.

File: _ont/ont-leox-lxt-010s-h.md [hunk 2]
```diff
@@ -46,7 +84,27 @@ GPON_PLOAM_PASSWD=AAAAAAAAAA
...
+
+to change the link mode use this command:
+
+```sh
+# /etc/scripts/flash get LAN_SDS_MODE
+LAN_SDS_MODE=1
+# /etc/scripts/flash set LAN_SDS_MODE 1
+```
+
+## Known Bugs
```
Comment (simonebortolin) [inline _ont/ont-leox-lxt-010s-h.md:103]: Known Bugs are (in all other pages) a first level header, use `#` inside `##.`

File: _ont/ont-leox-lxt-010s-h.md [hunk 3]
```diff
@@ -12,27 +12,65 @@ parent: Leox
...
+## Serial Console
+
+The stick has a TTL 3.3v UART console (115200bps - 8Bit) that can be accessed from the top surface. To accept TX line commands, GND of TTL adapter should be attached to the stick's shield:
+
+{% include image.html file="ont-leox-lxt-010s-h_ttl.jpg" alt="Leox LXT-010S-H TTL Pinout" caption="Leox LXT-010S-H TTL Pinout" %}
+
+## List firmware version
+
+- V3.3.4L3
+- V3.3.4L4rc1 (contains fix for 2.5GbE mode)
```
Comment (simonebortolin) [inline _ont/ont-leox-lxt-010s-h.md:59]: add this info:
`There is a new firmware avaliable from LeoLabs that fix this issue`
and move this info:
`contains fix for 2.5GbE mode
`with a  alert box like this:
`{% include alert.html content="Try PIN 10 or other GND PINs if it doesn't work with 14." alert="Note" icon="svg-warning" color="yellow" %}`

File: _ont/ont-leox-lxt-010s-h.md [hunk 4]
```diff
@@ -46,7 +84,27 @@ GPON_PLOAM_PASSWD=AAAAAAAAAA
 # /etc/scripts/flash set GPON_PLOAM_PASSWD AAAAAAAAAA
 ```
 
-# Known Bugs
+## LAN SDS Mode (with firmware V3.3.4L4rc1):
+
+|       mode           |             description              |
+| -------------------- | ------------------------------------ |
+| LAN_SDS_MODE = 1     | 1GbE with auto-neg on                |
```
Comment (simonebortolin) [inline _ont/ont-leox-lxt-010s-h.md:91]: if you know specify whether the other modes 2 to 6 exist or not

File: _ont/ont-leox-lxt-010s-h.md [hunk 5]
```diff
@@ -12,27 +12,65 @@ parent: Leox
...
+| mtd12 | 00300000 | 00001000  | "linux"         |
+| mtd13 | 004c0000 | 00001000  | "rootfs"        |
+
+This stick supports dual boot. 
+
+`k0` and `r0` respectively contain the kernel and firmware of the first image, `k1` and `r1` the kernel and the firmware of the second one
+
+## Serial Console
+
+The stick has a TTL 3.3v UART console (115200bps - 8Bit) that can be accessed from the top surface. To accept TX line commands, GND of TTL adapter should be attached to the stick's shield:
```
Comment (simonebortolin) [inline _ont/ont-leox-lxt-010s-h.md:52]: I do not remember whether this is the syntax already used on other pages or whether it is different. I don't think there is a capital B of bit on other pages.

**Review by simonebortolin — CHANGES_REQUESTED** [review]

**Comment by simonebortolin** [comment]
finally, will you create me a nice commit message that will take the place of all those you have already written?

**Comment by qatar2030** [comment]
i have LEOX GPON STICK LXT-010S-H and it was working great with mikrotik rb5009 and when i change mode from flash set LAN_SDS_MODE 1 to flash set LAN_SDS_MODE 8 the sfp stop working any help please

**Comment by simonebortolin** [comment]
> i have LEOX GPON STICK LXT-010S-H and it was working great with mikrotik rb5009 and when i change mode from flash set LAN_SDS_MODE 1 to flash set LAN_SDS_MODE 8 the sfp stop working any help please

Please open a discussion/issue

**Comment by stich86** [comment]
Which version of RouterOS and Firmware do you have?

The support was added by 7.3 or later.
Make sure that firmware (under System/RouterBOARD) is update otherwise 2.5Gbit// doesn't work

**Comment by qatar2030** [comment]
> Which version of RouterOS and Firmware do you have?
> 
> The support was added by 7.3 or later. Make sure that firmware (under System/RouterBOARD) is update otherwise 2.5Gbit// doesn't work

My SFP firmware version is V3.3.4L4rc1 (Fix 2.5GbE HiSGMII) and my Mikrotik version 7.11
Screenshot 2023-08-20 094654
Screenshot 2023-08-20 094711
and someone taled me that I have to have media convertor 2.5G instead 1G to change sfp mode to fiber
Screenshot 2023-08-20 095302

**Comment by simonebortolin** [comment]
Please open a NEW issue

**Pull request merged**